src/inlines.cpp

```diff
@@ -35,6 +35,9 @@
 // the stubs correctly at link time by itself
 #if !defined(__CYGWIN__)
 
+// Same as above for Apple ARM64 platforms
+#if !defined(__APPLE__) && !defined(__aarch64__)
```

Comment:
Might it be better to say:

```
#if !(defined(__APPLE__) && defined(__aarch64__))
```

so that the change only applies to ARM64 Macs?

Reply:
Thanks for pointing this out! It should have been a `||` instead of `&&`, that's what I get doing things in a rush and not paying attention. 

I have now changed the line as you suggest as it's definitely more readable.